LibTest/io/Stdout/Stdout_class_A01_t01.dart

```diff
@@ -11,7 +11,7 @@
 import "dart:io";
 import "../../../Utils/expect.dart";
 
-main() async {
+main() {
```

Comment:
Yes, that's good, we shouldn't completely eliminate testing the case where `main` does not have an explicit return type.